sfge/src/main/java/com/salesforce/apex/jorje/JorjeUtil.java

```diff
@@ -130,5 +135,14 @@ public static class JorjeCompilationException extends SfgeRuntimeException {
         }
     }
 
+    /**
+     * Increments log level of BaseApexLexer class from jorje jar
+     * to avoid printing info logs.
+     */
+    private static void incrementLogLevel() {
+        Logger jorjeLogger = Logger.getLogger(BaseApexLexer.class.getName());
+        jorjeLogger.setLevel(Level.WARNING);
```

Comment:
I'm unable to repro the info log line consistently and hence don't have a unit test.